shared/src/models/utils/tag-helpers.ts

```diff
@@ -38,3 +46,109 @@ export function createPatientTag(
         patientId
     );
 }
+
+export function createRadiogramTypeTag(
+    draftState: Mutable<ExerciseState>,
+    radiogramId: UUID
+): Tag {
+    const radiogram = getExerciseRadiogramById(draftState, radiogramId);
+    if (!radiogram) {
+        throw new Error(
+            `Expected to find radiogram with id ${radiogramId} while creating logs, but did not find it.`
+        );
+    }
```

Comment:
Again, this check cannot trigger because the `getExerciseRadiogramById` already checks for falsy values.